### ISSUE TITLE
Optional parameters not supported

### ISSUE THREAD

**Issue by ankurk91** [tightenco/ziggy#31]
Hi,

* ziggy : v0.2.2
* Laravel: v5.4.30

`route()` method throws error if [optional](https://laravel.com/docs/5.4/routing#parameters-optional-parameters) parameters not passed
```php
//routes/web.php
Route::get("/check/{id?}")->name("check")
```
```js
// `id` should be optional, next line will throw error
route("check")
// works when id is passed
route("check",{id:123})

// works when id is passed (master branch)
route("check",{"id?":123})
```

**Comment by DanielCoulbourne** [comment]
Good call. I'll try to get this fixed soon.

**Comment by DanielCoulbourne** [comment]
Closed by #33
